Backtesting/collectors/exchanges/exchange_history_collector/history_collector.py

```diff
@@ -34,11 +34,15 @@ class ExchangeHistoryDataCollector(collector.AbstractExchangeHistoryCollector):
 
     def __init__(self, config, exchange_name, tentacles_setup_config, symbols, time_frames,
                  use_all_available_timeframes=False,
-                 data_format=backtesting_enums.DataFormats.REGULAR_COLLECTOR_DATA):
+                 data_format=backtesting_enums.DataFormats.REGULAR_COLLECTOR_DATA,
+                 start_timestamp=None,
```

Comment:
I think we should prefer adding `start_timestamp` and `end_timestamp` to `ExchangeDataCollector`'s `__init__`

Reply:
Do you mean in the `__init__` function of `AbstractExchangeHistoryCollector` ?

Reply:
No the `ExchangeDataCollector` class from OctoBot-Backtesting